tern/__main__.py

```diff
@@ -208,6 +208,13 @@ def main():
                                help="Generate a report for the current state "
                                "of the filesystem. This is useful when "
                                "running Tern in a new namespace or VM.")
+    parser_report.add_argument('-ctx', '--with-context', metavar='FILES',
+                               nargs='+',
+                               help="When generating a report, take into "
+                               "account data collected from a previous run "
+                               "without including that data in the report. "
+                               "This is useful when running Tern multiple "
```

Comment:
Do you want to add anything here about how folks can string together multiple SBoMs or the fact that they must be ordered for Tern to read them or is this something that you are going to write out a longer explanation for in the README?

Reply:
Another question: Do you want to clarify that the reports provided should only be for one layer or can Tern handle reports that represent multiple layers?

Reply:
Good points. Let me update this message to be more clear.

Reply:
@nishakm looks perfect! LGTM